packages/augur-ui/src/modules/market/containers/market-orders-positions-table.ts

```diff
@@ -1,31 +1,26 @@
-import { connect } from "react-redux";
-import { withRouter } from "react-router-dom";
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
 
-import MarketOrdersPositionsTable from "modules/market/components/market-orders-positions-table/market-orders-positions-table";
-import { selectMarket } from "modules/markets/selectors/market";
-import {
-  MODAL_CLAIM_TRADING_PROCEEDS
-} from "modules/common/constants";
-import { selectCurrentTimestamp } from "store/select-state";
-import { CONTRACT_INTERVAL } from "modules/common/constants";
-import { updateModal } from "modules/modal/actions/update-modal";
-import { createBigNumber } from "utils/create-big-number";
-import { cancelAllOpenOrders } from "modules/orders/actions/cancel-order";
-import { selectUserFilledOrders } from "modules/orders/selectors/filled-orders";
-import getUserOpenOrders from "modules/orders/selectors/user-open-orders";
+import MarketOrdersPositionsTable from 'modules/market/components/market-orders-positions-table/market-orders-positions-table';
+import { selectMarket } from 'modules/markets/selectors/market';
+import { createBigNumber } from 'utils/create-big-number';
+import { cancelAllOpenOrders } from 'modules/orders/actions/cancel-order';
+import { selectUserFilledOrders } from 'modules/orders/selectors/filled-orders';
+import getUserOpenOrders from 'modules/orders/selectors/user-open-orders';
+import { ZERO } from 'modules/common/constants';
 
 const mapStateToProps = (state, ownProps) => {
   const market = ownProps.market || selectMarket(ownProps.marketId);
   let openOrders = getUserOpenOrders(market.id) || [];
 
   let canClaim = false;
-  if (market.finalizationTime) {
-    const endTimestamp = createBigNumber(market.finalizationTime).plus(
-      createBigNumber(CONTRACT_INTERVAL.CLAIM_PROCEEDS_WAIT_TIME)
-    );
-    const currentTimestamp = selectCurrentTimestamp(state);
-    const timeHasPassed = createBigNumber(currentTimestamp).minus(endTimestamp);
-    canClaim = timeHasPassed.toNumber() > 0;
+  if (
```

Comment:
Here is the other canClaim

Reply:
Ok, I'll be out of town tomorrow, but I'll try to work on this on Sunday.

Reply:
After looking at this code a bit more, I noticed `hasClaimableReturns` wasn't actually needed anymore, so `canClaim` was also not needed in this file. As a result, a helper method isn't needed either, so I just moved the code from the render method in `market-header-reporting.tsx` to the container `market-header-reporting.ts`

Reply:
Awesome!